ci/package.sh

```diff
@@ -68,47 +113,41 @@ do
 
                     if [ -d $stack_dir/image ]
                     then
-                        docker build -t $DOCKERHUB_ORG/$stack_id \
+                        docker build \
+                            --build-arg REPO_SLUG=$REPO_SLUG \
+                            --build-arg DOCKERHUB_ORG=$DOCKERHUB_ORG \
+                            --build-arg STACK_ID=$stack_id \
+                            --build-arg MAJOR_VERSION=$stack_version_major \
+                            --build-arg MINOR_VERSION=$stack_version_minor \
+                            --build-arg PATCH_VERSION=$stack_version_patch \
+                            -t $DOCKERHUB_ORG/$stack_id \
```

Comment:
I like this and will make it easier for stack creators to maintain stacks.

Reply:
I will add this specific piece into the other change I'm working on (as it's an isolated change to provide additional arguments to the stack.

Note that for these arguments to have any use.. the stack needs to build the templates. We could do this with volume mounts, but that is just as complicated in its way..